js/ui/viewSelector.js

```diff
@@ -205,9 +205,9 @@ const ViewSelector = new Lang.Class({
         Main.overview.show();
     },
 
-    show: function() {
+    show: function(viewPage) {
```

Comment:
I guess we're left with a broken commit here then, but perhaps it's not such a big deal. Something to take care of with the next rebase.